e2e-test/e2e/chaos/networkchaos/network_delay.go

```diff
@@ -100,6 +100,7 @@ func TestcaseNetworkDelay(
 	err := cli.Create(ctx, networkDelay.DeepCopy())
 	framework.ExpectNoError(err, "create network chaos error")
 
+	// nolint
```

Comment:
I wonder what does this `// nolint` ignore? And what are the standards that you choose which error to fix and which to ignore?